templates/base.html

```diff
@@ -2,20 +2,76 @@
 
 <!DOCTYPE html>
 <html lang="en-gb">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-<head>
+  <head>
     <title>{% block title %} {% endblock %}| Share Spots</title>
     <!-- Latest compiled and minified CSS -->
-</head>
-
-<body>
+  </head>
 
+  <body>
     {% block layout %} {% endblock %}
 
     <script src="{% static 'main.js' %}" type="module"></script>
-    <link href='https://fonts.googleapis.com/css?family=Monoton|Roboto' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Raleway:300,700,500' rel='stylesheet' type='text/css'>
-</body>
+    <link
```

Comment:
there is some weird spacing from this line onwards..

Reply:
The formatting is because I accidentally formatted with Prettier:
https://blog.codepen.io/2017/04/27/prettier-prettification/

I use it with most projects. Maybe you could watch the video above, and play with Prettier and have a think about wether it would be a good idea to use Prettier on Sharespots or not?

The easiest way to use Prettier is with this VSCode plugin:
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

 I accidentally had the editor.formatOnSave setting as true which is why there is all the reformatting in this pull request.

I could re-do this pull request without Prettier reformatting, or maybe we should be using Prettier in Sharespots for more readable code and better developer coding experience? Let me know what you think?

Reply:
I think last word here should be from @lili2311  whether we should use prettier on Sharespots or not :)

Reply:
I've started a branch to autoformat code using prettier (and beautify) here
https://github.com/PolyglotDevsLondon/sharespots/tree/chore/format-code